fetch-publication/src/test/java/no/unit/nva/fetch/FetchPublicationHandlerTest.java

```diff
@@ -29,44 +24,35 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 import static java.util.Collections.singletonMap;
-import static no.unit.nva.publication.FetchPublicationHandler.ACCESS_CONTROL_ALLOW_ORIGIN;
-import static no.unit.nva.publication.FetchPublicationHandler.ALLOWED_ORIGIN_ENV;
-import static no.unit.nva.publication.FetchPublicationHandler.API_HOST_ENV;
-import static no.unit.nva.publication.FetchPublicationHandler.API_SCHEME_ENV;
-import static no.unit.nva.publication.service.FetchResourceService.AUTHORIZATION;
+import static no.unit.nva.fetch.FetchPublicationHandler.ACCESS_CONTROL_ALLOW_ORIGIN;
+import static no.unit.nva.fetch.FetchPublicationHandler.ALLOWED_ORIGIN_ENV;
+import static no.unit.nva.service.impl.RestPublicationService.AUTHORIZATION;
 import static org.apache.http.HttpHeaders.CONTENT_TYPE;
 import static org.apache.http.HttpStatus.SC_BAD_GATEWAY;
 import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
 import static org.apache.http.HttpStatus.SC_INTERNAL_SERVER_ERROR;
 import static org.apache.http.HttpStatus.SC_NOT_FOUND;
 import static org.apache.http.HttpStatus.SC_OK;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
 public class FetchPublicationHandlerTest {
 
     public static final String SOME_API_KEY = "some api key";
     public static final String PATH_PARAMETERS = "pathParameters";
     public static final String HEADERS = "headers";
     public static final String IDENTIFIER = "identifier";
     public static final String IDENTIFIER_VALUE = "0ea0dd31-c202-4bff-8521-afd42b1ad8db";
-    public static final String RESOURCE_RESPONSE_JSON = "src/test/resources/resource_response.json";
-    public static final String EMPTY_RESPONSE_JSON = "src/test/resources/empty_response.json";
-    public static final String MISSING_FILE_JSON = "missing_file.json";
+    public static final String PUBLICATION_JSON = "src/test/resources/publication.json";
```

Comment:
We do not have to assume a certain structure for the code But do not do anything now. I am working on the nva-commons.
```
 Thread.currentThread().getContextClassLoader().getResourceAsStream("publication.json");
```